graylog2-web-interface/src/components/search/MessageDetailsTitle.jsx

```diff
@@ -25,7 +25,7 @@ const Title = styled.h3(({ theme }) => css`
   }
 
   .label {
-    font-size: calc(${theme.fonts.size.small} + 50%);
+    font-size: calc(${theme.fonts.size.h3} - 40%);
```

Comment:
```suggestion
    font-size: ${theme.fonts.size.tiny;
```

Reply:
![image](https://user-images.githubusercontent.com/122591/113144954-758a2400-91f3-11eb-91f1-40267d4a7609.png)

Reply:
I like it 👍